packages/tools/devtools/devtools-core/src/data-visualization/VisualSharedTreeTypes.ts

```diff
@@ -0,0 +1,44 @@
+/*!
```

Comment:
I would probably merge these in with the contents of `SharedTreeHelper`. That should be the only place they are used, and that way we don't have to export any of the types.

Reply:
And once everything else has stablized, it should be possible to remove these types altogether and update your visualization functionality to generate `VisualTree`s directly. This would be a small perf win, and would reduce the number of concepts devs have to learn to know what's going on here.

That said, I would definitely wait to make this change until after everything else here has stabilized. And it could definitely be done in a separate PR, though we should track it via an ADO task in that case.

Reply:
Have we made a work item for this?

Reply:
I will create one for this.